gpii/node_modules/flowManager/src/CloudBasedFlowManager.js

```diff
@@ -44,6 +46,9 @@ fluid.defaults("gpii.flowManager.cloudBased", {
             defaultGradeNames: "gpii.flowManager.cloudBased.cloudStatus"
         }
     },
+    // File containing the full sha256 of the revision of the repository for
+    // this CBFM, e.g., { "sha256": "86a83d2f93a6f8f954a4fef618ca6aea1399c980" }
+    gpiiRevisionPath: "%gpii-universal/gpii-revision.json",
```

Comment:
This file path should be distributed from [the flow manager cloud config](https://github.com/GPII/universal/blob/master/gpii/node_modules/flowManager/configs/gpii.flowManager.config.cloud.base.json5).